dotnet/api/Api.sln

```diff
@@ -17,8 +17,6 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Workers.Hangfire", "Infrast
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Presentation", "Presentation", "{2BCBBA5C-033C-4CF6-90A4-A21CCEA15E0C}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Cli", "Presentation\Cli\Cli.csproj", "{706B0EAB-ECDE-45D7-A31D-AB4C85F7A2A2}"
```

Comment:
Thought - instead of killing this project entirely, how do we feel about keeping it in here? While the first use-case was simply test database migration, I could see it being used for seeding test data or other devops-related activity that interacting directly with the .NET Core app would ease.

Not sure how everyone else feels about it. I'm not suggesting we leave in commented out test database code, but maybe just a sample command or two to show how to stand one up.

Reply:
Yeah, lets leave the CLI project - while some of our current projects have not leveraged it much, the hope is (realize things move slowly sometimes) is we can further leverage this project moving forward. Examples such as robust database seeding strategies, and more specific c# devops related tasks like @brandongregoryscott suggested.

Reply:
Roger that. I removed it because we've been leaning on the `and-cli` for these types of tasks.